packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -204,25 +207,26 @@ export type SupportedCardConfigType = CardConfigItemV1;
 
 export type UserState = Readonly<
   | {
-      status: 'loggedOut';
-    }
+    status: 'loggedOut';
+  }
   | {
-      status: 'authenticated';
-      authorization: Authorization;
-      roles: Roles;
-    }
+    status: 'authenticated';
+    authorization: Authorization;
+    roles: Roles;
+  }
   | {
-      status: 'expired';
-      authorization: Authorization;
-    }
+    status: 'expired';
+    authorization: Authorization;
+  }
 >;
 
 export class NotifiFrontendClient {
   constructor(
     private _configuration: NotifiFrontendConfiguration,
     private _service: NotifiService,
     private _storage: NotifiStorage,
-  ) {}
+    private _wsUrl: string
```

Comment:
nit: _wssUrl

Reply:
I would not put the wsUrl here since it is the constant. 
Now all the endpoint related operations are in `notifi-graphql`'s `NotifiService`. Do you think would it be a good idea to move it here? So it will be similar like below:
``` typescript
import { createClient as createWsClient } from 'graphql-ws';
// ...
export const new NotifiService = (config: NotifiFrontendConfiguration) => {
 //  ...
  const wsUrl = envUrl(/* TODO, See my anohter comment */);
  const wsClient = createWsClient({ url });
  return new NotifiService(client, wsClient); // The endpoint operations to be migrated to 'notifi-graphql'
};
```